### PR TITLE
Migrate proximity to sensor entities

### PR DESCRIPTION
**Pull request by mib1185**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/101497
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

File: source/_integrations/proximity.markdown [hunk 1]
```diff
@@ -20,23 +20,31 @@ Some examples of its use include:
...
-  - `yd`
-  - `ft`
-- `nearest`: The device or person which is nearest to the zone
+## Sensors
+
+The following sensor entities will be created
+
+### Distance
+
+For each tracked device or person, the distance from the monitored zone in an unit depending on your [Home Assistant Unit System](/docs/configuration/basic) selection.
```
Comment (MartinHjelmare) [inline source/_integrations/proximity.markdown]:
```suggestion
For each tracked device or person, the distance from the monitored zone in a unit depending on your [Home Assistant Unit System](/docs/configuration/basic) selection.
```

Comment (MartinHjelmare) [reply source/_integrations/proximity.markdown]: It's weird to not have a verb in these sentences. Not sure what would be better.

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]

File: source/_integrations/proximity.markdown (hunk 1 above)
Comment (mib1185) [reply source/_integrations/proximity.markdown]:
```suggestion
For each tracked device or person, a sensor showing the distance from the monitored zone in a unit depending on your [Home Assistant Unit System](/docs/configuration/basic) selection is created.
```

what about this one?

Comment (MartinHjelmare) [reply source/_integrations/proximity.markdown]: Yes, that's good. But replace an with a.

File: source/_integrations/proximity.markdown [hunk 2]
```diff
@@ -20,23 +20,31 @@ Some examples of its use include:
...
-- `unit_of_measurement`: Measurement of distance. Values are:
-  - `km`
-  - `m`
-  - `mi`
-  - `yd`
-  - `ft`
-- `nearest`: The device or person which is nearest to the zone
+## Sensors
+
+The following sensor entities will be created
```
Comment (MartinHjelmare) [inline source/_integrations/proximity.markdown]:
```suggestion
The following sensor entities will be created.
```

File: source/_integrations/proximity.markdown [hunk 3]
```diff
@@ -20,23 +20,31 @@ Some examples of its use include:
...
+
+### Distance
+
+For each tracked device or person, the distance from the monitored zone in an unit depending on your [Home Assistant Unit System](/docs/configuration/basic) selection.
+You can use the [Min/Max](/integrations/min_max) integration to determine the nearest and furthest distance.
+
+### Direction of travel
+
+For each tracked device or person, the direction of travel to or from the monitored zone. Values are:
```
Comment (mib1185) [inline source/_integrations/proximity.markdown]:
```suggestion
For each tracked device or person, a Sensor showing the direction of travel to or from the monitored zone is created. Possible states are:
```

File: source/_integrations/proximity.markdown [hunk 4]
```diff
@@ -20,23 +20,31 @@ Some examples of its use include:
...
+- `not_set`
+- `arrived`
+- `towards`
+- `away_from`
+- `unknown`
+- `stationary`
+
+### Nearest device
+
+The device or person which is nearest to the zone
```
Comment (mib1185) [inline source/_integrations/proximity.markdown]:
```suggestion
A Sensor showing the device or person which is nearest to the zone is created.
```

File: source/_integrations/proximity.markdown [hunk 5]
```diff
@@ -20,23 +20,31 @@ Some examples of its use include:
...
+The following sensor entities will be created.
+
+### Distance
+
+For each tracked device or person, a sensor showing the distance from the monitored zone in a unit depending on your [Home Assistant Unit System](/docs/configuration/basic) selection is created.
+You can use the [Min/Max](/integrations/min_max) integration to determine the nearest and furthest distance.
+
+### Direction of travel
+
+For each tracked device or person, a Sensor showing the direction of travel to or from the monitored zone is created. Possible states are:
```
Comment (mib1185) [inline source/_integrations/proximity.markdown]:
```suggestion
For each tracked device or person, a sensor showing the direction of travel to or from the monitored zone is created. Possible states are:
```

File: source/_integrations/proximity.markdown [hunk 6]
```diff
@@ -20,23 +20,31 @@ Some examples of its use include:
...
+- `not_set`
+- `arrived`
+- `towards`
+- `away_from`
+- `unknown`
+- `stationary`
+
+### Nearest device
+
+A Sensor showing the device or person which is nearest to the zone is created.
```
Comment (mib1185) [inline source/_integrations/proximity.markdown]:
```suggestion
A sensor showing the device or person which is nearest to the monitored zone is created.
```

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

File: source/_integrations/proximity.markdown [hunk 7]
```diff
@@ -20,23 +20,31 @@ Some examples of its use include:
...
+
+### Direction of travel
+
+For each tracked device or person, a sensor showing the direction of travel to or from the monitored zone is created. Possible states are:
+
+- `not_set`
+- `arrived`
+- `towards`
+- `away_from`
+- `unknown`
```
Comment (MartinHjelmare) [inline source/_integrations/proximity.markdown]: The order here seems a bit awkward.

Did we keep `not_set`?

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]
Noticed the enum names.

File: source/_integrations/proximity.markdown [hunk 8]
```diff
@@ -20,23 +20,31 @@ Some examples of its use include:
...
+- `stationary`
```
Comment (mib1185) [inline source/_integrations/proximity.markdown]:
```suggestion
- `arrived`
- `away_from`
- `stationary`
- `towards`
- `unknown`
```

File: source/_integrations/proximity.markdown (hunk 7 above)
Comment (mib1185) [reply source/_integrations/proximity.markdown]: you're right - we have removed `not_set` 👍

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

**Pull request merged**